### PR TITLE
replacing flex

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
- [DEMO LINK](https://Anton-Zhytonbaiev.github.io/layout_catalog_grid/)

### CODE REVIEW

**Comment by OlyaZaiets** [comment]
Did a good job, but need to do some modification. 
Doesn't develop main.scss, normalize.scss, and reset files.

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,10 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
```
Comment (OlyaZaiets) [inline src/styles/_catalog.scss:3]: should we save this code?

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
 
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    & {
```
Comment (OlyaZaiets) [inline src/styles/_catalog.scss:11]: We can delete this row

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
+    }
+  }
+
+  @media (max-width: 488px) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
+
   &__card {
```
Comment (OlyaZaiets) [inline src/styles/_catalog.scss:28]: Don't need pseudo-class here

Comment (OlyaZaiets) [reply src/styles/_catalog.scss:28]: Big distance between rows 
image

File: src/styles/_catalog.scss [hunk 4]
```diff
@@ -1,10 +1,30 @@
...
   margin: 0 auto;
 
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+
+  @media (max-width: 1024px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
```
Comment (OlyaZaiets) [inline src/styles/_catalog.scss:12]: According to the task,  four columns start at 1024px, but you have 3 columns. The same problem with 768, 488, and less than 488 px
image

File: src/styles/_catalog.scss [hunk 5]
```diff
@@ -1,10 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
 
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
```
Comment (Anna-Tkachenko) [inline src/styles/_catalog.scss:7]: create separate variable for the card width value

File: src/styles/_catalog.scss (hunk 2 above)
Comment (Anna-Tkachenko) [reply src/styles/_catalog.scss:11]: agree, fix @OlyaZaiets comment, this line is redundant

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**